app/src/main/java/com/samco/trackandgraph/ui/DurationInputView.kt

```diff
@@ -49,7 +51,11 @@ class DurationInputView : FrameLayout {
             }
             return@setOnEditorActionListener false
         }
-        binding.hoursInput.requestFocus()
+        binding.hoursInput.focusAndShowKeyboard()
+    }
+
+    override fun requestFocus(direction: Int, previouslyFocusedRect: Rect?): Boolean {
+        return binding.hoursInput.requestFocus()
```

Comment:
I swapped this for 

        binding.hoursInput.focusAndShowKeyboard()
        return true 
       
because it seems to work better.